diff --git "a/private-source/P02_tables_current_R734_extract.tex" "b/private-source/R753_P02_tables_live_extract.tex"
index abfe8c8..a0a4e68 100644
--- "a/private-source/P02_tables_current_R734_extract.tex"	
+++ "b/private-source/R753_P02_tables_live_extract.tex"	
@@ -83,4 +83,4 @@
 \vspace{-0.2em}
 \begin{center}\footnotesize (Die Zahlen der obersten Horizontalreihe bezeichnen den Grad in den $x$, die Zahlen der ersten Vertikalreihe den Grad in den $u$.)\end{center}
 \endgroup
-\end{landscape}
\ No newline at end of file
+\end{landscape}
